getting_hired/applying_and_interviewing/preparing_to_interview_and_interviewing.md

```diff
@@ -26,9 +26,9 @@ You'll probably be asked about some of the technical things you've listed on you
 
 FINAL NOTE -- this is not one-size-fits-all and many companies skip the light stuff and dive right into a technical screen so you've got to be prepared just in case!  The Coding Horror link below is more descriptive about that case.
 
-* [Mastering the phone screen by Monster](http://career-advice.monster.com/job-interview/interview-preparation/mastering-the-phone-interview/article.aspx)
```

Comment:
Not having issues with the Monster link. Have you given it another try on another device possibly? The second link however does seem to be broken. So if we could keep the old link for the Mastering the phone screen by Monster link and just replace the other link I can go ahead and approve this.

Reply:
It might be worth still updating this link to what you get redirected to: `https://www.monster.com/career-advice/article/mastering-the-phone-interview`

Reply:
That works as well. Are you still interested in doing this @lopezac